lib/mayaUsd/render/vp2RenderDelegate/basisCurves.cpp

```diff
@@ -1666,6 +1667,10 @@ MHWRender::MRenderItem* HdVP2BasisCurves::_CreateWireRenderItem(const MString& n
     renderItem->setShader(_delegate->Get3dSolidShader(kOpaqueGray));
     renderItem->setSelectionMask(MSelectionMask::kSelectCurves);
 
+#if MAYA_API_VERSION >= 20210000
```

Comment:
Hey all, is it possible that we might need to use a higher API version number here? We're continuing to build against PR 121 at the moment while we get our licenses sorted out for the newly released PR 122, but compilation against PR 121 seems to be failing with `setObjectTypeExclusionFlag()` not being found.

Reply:
For beta releases, we only support the last release, i.e. PR122.